pinot-spi/src/main/java/org/apache/pinot/spi/config/table/RoutingConfig.java

```diff
@@ -27,6 +27,7 @@
 
 public class RoutingConfig extends BaseJsonConfig {
   public static final String PARTITION_SEGMENT_PRUNER_TYPE = "partition";
+  public static final String TIME_RANGE_SEGMENT_PRUNER_TYPE = "timeRange";
```

Comment:
Let's name it `time` to be consistent with `partition`. Also it works not only on time range, but also point time lookup
```suggestion
  public static final String TIME_SEGMENT_PRUNER_TYPE = "time";
```